apps/client/src/app/import/import.component.ts

```diff
@@ -44,6 +35,8 @@ class ImportFileModel {
   public vsBundle?: Bundle;
   public message: string;
   public status: 'add'|'update'|'unauthorized'|'pending'|'unknown' = 'pending';
+  public multiple = false;
```

Comment:
Consider if other developers know what  "multiple" represents. Maybe consider name `multipleIgs`...

Reply:
Did that